apps/single-view/src/Utils/formatDates.test.tsx

```diff
@@ -2,8 +2,8 @@ import { formatDateOfBirth, formatDate } from "./formatDates";
 
 describe("formatDateOfBirth function", () => {
   it("should display a person's date of birth in the format, DD/MM/YYYY", () => {
-    const result = formatDateOfBirth("1990-12-01");
-    expect(result).toEqual("01/12/1990");
+    const result = formatDateOfBirth("1980-02-01T00:00:00Z");
+    expect(result).toEqual("01/02/1980");
```

Comment:
could we also keep the previous tests so that we are confident that both date formats are parsed correctly?